### PR TITLE
docs/update dev cycle

### PR DESCRIPTION
**Pull request by mavogel**
please sb new test the steps and enhance if necessary :)

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -84,8 +145,29 @@ ROOT_MAIL_DOMAIN=... \
 
 #### Create a pull request
 
-Creating a PR will trigger the [build job](tests/build.yaml) and run the test suite (if PR creator is in the allow-list).
+Creating a PR will trigger the [build job](tests/build.yaml) and run the test suite (if PR creator is in the `AllowedGitHubAccountIDs` as mentioned above).
+
+## Releasing
+
+We are using the [semantic-release-action](https://github.com/cycjimmy/semantic-release-action), so the plugin determines which version it will cut based on the semantic commit messages (see the format [here](https://github.com/semantic-release/semantic-release)) from the last release until now.
```
Comment (aperov9) [inline DEVELOPMENT.md:152]: please add that we are using conventional commits https://www.conventionalcommits.org/

Comment (aperov9) [reply DEVELOPMENT.md:152]: *was decided in joint session

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -65,15 +117,24 @@ ACCOUNT_FACTORY_ACCOUNT_ID=... \
 
 #### Login to the test environment
 
+This uses `firefox` and logs you in to the test account. You get the `AWS_ACCOUNT_ID` as follows
+1. In the GitHub PR click in the *Details* link of the `AWS CodeBuild BuildBatch eu-west-1` check
+2. Click on the `uuid` of the `Build run`, where you find the `arn` of the `OVMCrossAccountRole`, which contains the AccountID, where the fresh superwerker installation runs.
+3. If the build logs are not present any more due to the retention time, then take the `Build number` and look for the details in the [BuildAndTestProject](https://eu-west-1.console.aws.amazon.com/codesuite/codebuild/projects?region=eu-west-1) codebuild project
+ 
 ```bash
 SUPERWERKER_REGION=uk-east-1 \
```
Comment (aperov9) [inline DEVELOPMENT.md:126]: typo us-east-1

**Review by aperov9 — CHANGES_REQUESTED** [review]

File: DEVELOPMENT.md (hunk 2 above)
Comment (mavogel) [reply DEVELOPMENT.md:126]: is intentional so people have to adapt it after copy-pasting

File: DEVELOPMENT.md (hunk 1 above)
Comment (mavogel) [reply DEVELOPMENT.md:157]: yes will add the link

**Review by aperov9 — APPROVED** [review]
LGTO

**Pull request merged**